src/XMakeBuildEngine/UnitTests/BackEnd/MSBuild_Tests.cs

```diff
@@ -1609,5 +1605,41 @@ public void TargetNameIsCaseInsensitive()
                 File.Delete(projectFile2);
             }
         }
+
+        [Fact]
+        public void ProjectFileWithoutNamespaceBuilds()
+        {
+            string projectFile1 = ObjectModelHelpers.CreateTempFileOnDisk(@"
+                <Project>
+                    <Target Name=`Build` Outputs=`foo.out` />
+                </Project>
+                ");
+
+            string projectFile2 = ObjectModelHelpers.CreateTempFileOnDisk(@"
+                <Project>                  
+                    <Target Name=`t`>
+                        <MSBuild Projects=`" + projectFile1 + @"` Targets=`Build`>
+                            <Output TaskParameter=`TargetOutputs` ItemName=`out`/>
+                        </MSBuild>
+                        <Message Text=`[@(out)]`/>
+                    </Target>	
+                </Project>
+                ");
+
+            try
+            {
+                Project project = new Project(projectFile2);
+                MockLogger logger = new MockLogger();
+
+                project.Build(logger);
```

Comment:
The next assert would catch an issue here, but good call. I added that on the test I copied this from as well :).

---
In reply to: [80769718](https://github.com/Microsoft/msbuild/pull/1089#discussion_r80769718) [](ancestors = 80769718)